appservice/src/deploy/getDeployFsPath.ts

```diff
@@ -81,3 +84,8 @@ async function appendDeploySubpathSetting(targetPath: string, extensionPrefix: s
 
     return targetPath;
 }
+
+export type deployPaths = {
```

Comment:
nit: `export type deployPaths` just looks wrong to me and I'm not entirely sure why. A few thoughts:
1. Can we pick a pattern for interfaces vs. types? I guess I'd prefer interfaces since I feel like we use that more often, but I couldn't find any conclusive docs saying they're better so 🤷‍♂️
1. Should this start with a lowercase letter or uppercase? I feel like a type is more similar to an interface or class (which uses uppercase) than a const (which uses lowercase)